config/locales/en.yml

```diff
@@ -90,14 +90,13 @@ en:
 
   secondary_content_links:
     guidance_markdown: |
-      ##What is secondary content?
-
       Secondary content is content which is not linked to in the step by step, but has the step by step sidebar showing on it.
 
       We use it for content which:
 
-      - could help you complete a task but is not the main piece of content included in the step by step
+      - could help you complete a task but is not the main piece of content included in the step by step.
```

Comment:
Do you mean for the list item below to have a full stop too? Our styleguide tends to not have the periods at the end of list items: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps

Reply:
Good spot. I've just pasted from the document without looking a the diff. Removed the full stop.